sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventNames.java

```diff
@@ -300,6 +392,13 @@
      */
     public static final String MEDIA_JOB_STATE_CHANGE = "Microsoft.Media.JobStateChange";
 
+    /**
+     * Channel Archive heartbeat event data. Schema of the data property of an EventGridEvent for a
+     * Microsoft.Media.LiveEventChannelArchiveHeartbeatEventData event.
+     */
+    public static final String MEDIA_LIVE_EVENT_CHANNEL_ARCHIVE_HEARTBEAT =
+        "Microsoft.Media.LiveEventChannelArchiveHeartbeatEventData";
```

Comment:
Suffix LiveEventChannelArchiveHeartbeat**EventData**, the EventData can be removed?

Reply:
Yeah I think this was fixed in a later commit. Can you make sure you are using the latest SHA?

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/24482/files#diff-d116ed60612e9aae51f2b7fe7b2de9482e764e38fcf7010003b4f36746b11ad7R500

Regenerated with lastest SHA, but it still contains a EventData suffix. It is a new class.